### PR TITLE
update package extension naming docs

### PR DESCRIPTION
**Pull request by aplavin**
As discussed in https://github.com/JuliaLang/Pkg.jl/pull/3600, Pkg output always displays the parent package name next to the extension name. The naming from the current pkg docs example (`PlottingContourExt`) makes Pkg output super verbose, like `Plotting -> PlottingContourExt`. Especially for longer package names, it can easily exceed typical line length.

Here, I update the docs to show shorter nonredundant naming, `ContourExt` (leading to `Plotting -> ContourExt` output). Many packages already follow this short naming, and presumably more will do if it becomes the official recommended way.

### CODE REVIEW

**Review by IanButterworth — APPROVED** [review]
LGTM but will hold off merging for @KristofferC's review

**Comment by IanButterworth** [comment]
I'll merge tomorrow if there are no objections.

**Comment by KristofferC** [comment]
Fine by me

**Pull request merged**
